src/components/showcase/showcase.html

```diff
@@ -12,34 +12,39 @@
     </header>
     <content class="oui-showcase__main">
         <div class="oui-showcase__content">
-            <nav class="oui-list oui-list_nav oui-showcase__sidenav" role="complementary">
-                <ul class="oui-list__items">
-                    <li class="oui-list__group"
-                        ng-repeat="groupKey in $ctrl.secondLevelsChildren[$ctrl.currentSecondLevelStateName].groupsOrder track by $index">
-                        <span class="oui-list__header"
-                            ng-if="groupKey === 'undefined'"
-                            ng-bind="$ctrl.secondLevelsChildren[$ctrl.currentSecondLevelStateName].name">
-                        </span>
-                        <span class="oui-list__header"
-                            ng-if="groupKey !== 'undefined'"
-                            ng-bind="$ctrl.secondLevelsChildren[$ctrl.currentSecondLevelStateName].groups[groupKey].name || groupKey">
-                        </span>
-                        <ul class="oui-list__sub-items">
-                            <li class="oui-list__sub-item"
-                                ng-repeat="child in $ctrl.secondLevelsChildren[$ctrl.currentSecondLevelStateName].children[groupKey] track by child.state"
-                                ui-sref-active="oui-list__item_current">
-                                <a class="oui-list__link"
-                                    ng-bind="child.name"
-                                    ui-sref="{{child.state}}">
-                                </a>
-                            </li>
-                            <li class="oui-list__item"
-                                ng-if="!$ctrl.secondLevelsChildren[$ctrl.currentSecondLevelStateName].groupsOrder.length">
-                                No children state found under the state {{ $ctrl.currentSecondLevelStateName }}.
-                            </li>
-                        </ul>
-                    </li>
-                </ul>
+            <nav class="oui-sidebar oui-showcase__sidenav" role="complementary">
+                <div class="oui-sidebar-menu">
+                    <ul class="oui-sidebar-list">
+                        <li class="oui-sidebar-list__item"
+                            ng-repeat="groupKey in $ctrl.secondLevelsChildren[$ctrl.currentSecondLevelStateName].groupsOrder track by $index"
+                            ui-sref-active="oui-sidebar-list__item_active">
+                            <button class="oui-sidebar-link" type="button"
+                                ng-if="groupKey === 'undefined'"
```

Comment:
`groupKey === undefined` ?

Reply:
`groupKey === "undefined"`, not really the same.
But I'll check if that cannot be simplified (legacy code)